draft-illyes-cbcp.md

```diff
@@ -89,7 +101,24 @@ standard signals defined by
 [RFC9110](https://www.rfc-editor.org/rfc/rfc9110#name-server-error-5xx), preferably also
 additional heuristics such as relative response time of the server.
 
+As such, crawlers should log already visited URL and how many requests they sent to a resource 
+as well as the respective HTTP status codes in the responses,
+especially if error occur, to avoid repeatedly crawling the same source.
+
+Generally, crawlers should avoid multiple concurrent requests to the same ressources
+and use a crawling rate that is no higher than 1 request per second or as specified in
+the REP protocol. Further, ressources should not be re-crawled more often than every 24 hours.
+Ideally crawlers should limit the crawling depth and number of requests
+per ressource to avoid loops.
+
+Crwalers should not attempt to access restricted ressources that requires a login, used CAPTCHAs, or are behind paywalls.
```

Comment:
We specifically work with publishers to crawl behind paywalls: https://developers.google.com/search/docs/appearance/structured-data/paywalled-content

Reply:
Okay, we can add a clause like "if not explicitly agreed with the publishers". Would that work?

Or maybe we can even change "should not attempt to access" to "should not try to circumvent" or something? Would that make sense?

Reply:
Yeah, perhaps something like "should not try to circumvent authentication"?

Reply:
Is this okay?
```suggestion
Crawlers should not try to circumvent authentication or other access restrictions e.g. when a login is required, CAPTCHAs are used, or the content is behind a paywall, except if explicitly negotiated with the website owner.
```